app/database/models/server/channel.ts

```diff
@@ -131,6 +136,23 @@ export default class ChannelModel extends Model {
     /** settings: User specific settings/preferences for this channel */
     @immutableRelation(MY_CHANNEL_SETTINGS, 'id') settings!: Relation<MyChannelSettingsModel>;
 
+    /** Experimental Ideas */
+    // Has it got unread messages
+    @lazy isUnread = this.membership.observe().pipe(
+        switchMap((membership) => of(membership!.isUnread)),
+        distinctUntilChanged(),
+    );
+
+    // What should the badge count be
+    @lazy badgeCount = this.membership.observe().pipe(
+        switchMap((membership) => of(membership!.mentionsCount)),
+    );
+
+    // When the last post was posted
+    @lazy lastPostAt = this.membership.observe().pipe(
+        switchMap((membership) => of(membership!.lastPostAt)),
+    );
+
```

Comment:
After the changes we made where you can access `myChannel` this is no longer needed

```suggestion
```